css/styles.css

```diff
@@ -138,46 +140,49 @@
 .portfolio {
     background: #303441;
     border-bottom: 5px solid #373b4a;
+    padding-bottom: 75px;
+    padding-top: 38px;
+}
+
+.portfolio-main-content {
+    font-family: latoLight;
+    font-size: 18px;
+    padding-top: 17px;
 }
 
-.btn {
+.portfolio-buttons {
     margin-right: 10px;
     color: #606681;
 }
 
-.btn-outline-secondary:hover {
+.portfolio-buttons:hover {
     border: 1px solid white;
     background: #303441;
 }
 
 .portfolio-images {
-    margin-bottom: 96px;
-    padding: 22px;
+    margin: 22px 2px 22px 6px;
 }
 
-.col-sm-3 {
+.portfolio-image-grid {
     margin-bottom: 15px;
 }
 
 .about-us {
-    padding-top: 26px;
-    padding-left: 39px;
-    padding-right: 28px;
+    padding: 54px 28px 60px 33px;
 }
 
 .about-icons {
-    margin-bottom: 60px;
+    padding-bottom: 20px;
 }
 
 .person {
-    margin-top: 17px;
-    margin-bottom: 13px;
+    padding-top: 28px;
+    padding-bottom: 10px;
 }
 
 .contacts {
-    padding-top: 26px;
-    padding-left: 39px;
-    padding-right: 28px;
+    padding: 55px 20px 105px 36px;
 }
```

Comment:
@Chem-e not sure why you are using different padding for each section. To make it consistent you should only have top and bottom padding and this should probably be the same for all the sections.